File: random-album-cover-mastodon-p_ljC2d9V/spotify_get_random_album/entry.py
import requests

def handler(pd: "pipedream"):
  token = f'{pd.inputs["spotify"]["$auth"]["oauth_access_token"]}'
  authorization = f'Bearer {token}'
  headers = {"Authorization": authorization}

  params = {
    "type":"album",
    "q": pd.steps["create_search_string"]["$return_value"],
    "offset":pd.steps["select_offset"]["$return_value"]
  }

  r = requests.get('https://api.spotify.com/v1/search', params=params, headers=headers)
  # Export the data for use in future steps
  return r.json()

File: remove-backgrounds-at-scale-p_gYC1GJp/get_dropbox_upload_url/entry.js
import { axios } from "@pipedream/platform"
export default defineComponent({
  props: {
    dropbox: {
      type: "app",
      app: "dropbox",
    }
  },
  async run({steps, $}) {
    
    const data = {
      "commit_info":{
        "path": steps.generate_output_path.$return_value
      }
    };
    
    return await axios($, {
      method: "post",
      url: `https://api.dropboxapi.com/2/files/get_temporary_upload_link`,
      headers: {
        Authorization: `Bearer ${this.dropbox.$auth.oauth_access_token}`,
        "Content-Type": `application/json`,
      },
      data,
    })
  },
})
